packages/sui-ssr/archive/index.js

```diff
@@ -1,15 +1,22 @@
 const path = require('path')
 const fs = require('fs')
 const archiver = require('archiver')
-
+const program = require('commander')
+const authDefinitionBuilder = require('./authDefinitionBuilder')
 module.exports = ({outputZipPath, pkg}) =>
   new Promise((resolve, reject) => {
+    let authVariableDefinition = program.auth
+      ? authDefinitionBuilder(program.auth.split(':'))
+      : ''
     const output = fs.createWriteStream(outputZipPath)
     const archive = archiver('zip', {
       zlib: {level: 9}
     })
 
-    output.on('close', resolve)
+    output.on('close', () => {
+      console.log(' -> Success ✅'.green)
```

Comment:
Maybe, could be interesting have the size of the file in bytes, like this --> 

```javascript
console.log(archive.pointer() + ' total bytes');
```
?????

Reply:
sure! I'll do that and if someone thing that doesn't proceed I'll delete it.